homeassistant/components/easyenergy/__init__.py

```diff
@@ -25,6 +112,41 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+
+    async def get_gas_prices(call: ServiceCall) -> ServiceResponse:
```

Comment:
Can we use partial instead of all these wrapper methods?

Reply:
I need some concrete examples of how this should be used within the integration.

Reply:
See my last review